test/AddOption/multi-arg.py

```diff
@@ -36,7 +36,8 @@
 test.write(
     'SConstruct',
     """\
-env = Environment()
+DefaultEnvironment(tools=[])
+env = Environment(tools=[])
```

Comment:
why not leave the Environment call out entirely if env is never used? which I'm guessing is the case for this test, of course the diff doesn't show every line